main.go

```diff
@@ -558,6 +567,11 @@ func main() {
 		log.Warnf("Export UITest Artifacts (export_uitest_artifacts) turned on, but Xcode version >= 11. The test result bundle structure changed in Xcode 11 it does not contain TestSummaries.plist and Attachments directly, nothing to export.")
 	}
 
+	if simulatorDebug != never && xcodeMajorVersion < 10 {
+		log.Warnf("Collecting Simulator diagnostics is available with Xcode version 10 or newer")
```

Comment:
Collecting Simulator diagnostics is not available below Xcode version 10, current Xcode version: %s